ui/app/AppLayouts/Onboarding2/OnboardingFlow.qml

```diff
@@ -533,4 +548,16 @@ OnboardingStackView {
             onLinkActivated: (link) => root.linkActivated(link)
         }
     }
+
+    Component {
+        id: reencryptingDatabaseScreen
```

Comment:
The back button should be suppressed on that page probably, by setting the marker property.